src/fides/api/models/privacy_experience.py

```diff
@@ -72,6 +81,16 @@ class PrivacyExperienceConfigBase:
 
     dismissable = Column(Boolean)
 
+    show_layer1_notices = Column(Boolean, nullable=True, default=False)
+
+    @declared_attr
+    def layer1_button_options(cls) -> Column:
+        return Column(
+            EnumColumn(Layer1ButtonOption),
+            nullable=True,
+            index=False,
+        )
```

Comment:
What's the reason for making this a `declared_attr`?

Reply:
This is similar to how we declare `ComponentType(Enum)`. Basically using this instead of formalizing it in the DB allows for more flexibility with future enum changes where we won't need a whole migration.